File: Prog1part4.java
/*
 * Name: Travis Bittner
 * Email: travis.bittner@und.edu
 * 
 * Simple Java program that uses Scanner class to get user input and store in an array of ints.
 * Uses a single println statement to display the sum of coins multiplied by their respective denomination.
 * Uses DecimalFormat object to format to 2 decimals places.
 * 
 * Command line args parameter because GBD yelled about it. 
 */
import java.util.Scanner;
import java.text.*;


public class Prog1part4 {
    public static void main(String[] args) {
        int[] coins = new int[4];
        
        DecimalFormat decFormat = new DecimalFormat("#0.00");
        Scanner sc = new Scanner(System.in);
        
        System.out.println("Quarters: ");
        coins[0] = sc.nextInt();
        
        System.out.println("Dimes: ");
        coins[1] = sc.nextInt();
        
        System.out.println("Nickels: ");
        coins[2] = sc.nextInt();
        
        System.out.println("Pennies: ");
        coins[3] = sc.nextInt();
        
        System.out.println("Amount: $" + decFormat.format(((coins[0] * 0.25)
                            +(coins[1] * 0.10)
                            +(coins[2] * 0.05)
                            +(coins[3] * 0.01))));
        
    }
}

File: Prog1part4.cpp
/*
Name: Travis Bittner
Email: travis.bittner@und.edu

Simple C++ program that asks user for # of specific denominations. 
Uses an int array instead of 4 separate variables. 
Displays sum of coins multiplied by their respective denomination.

*/
#include <iostream>
#include <iomanip>

int main() {
	std::cout << setprecision(2) << fixed;
	
	int coins[4] = { 0 };

	std::cout << "Quarters: ";
	std::cin >> coins[0];

	std::cout << "Dimes: ";
	std::cin >> coins[1];

	std::cout << "Nickels: ";
	std::cin >> coins[2];

	std::cout << "Pennies: ";
	std::cin >> coins[3];

	//Sum the array of coins for total
	std::cout << "Amount: $" << 
		  (coins[0] * 0.25)
		+ (coins[1] * 0.10)
		+ (coins[2] * 0.05)
		+ (coins[3] * 0.01);

}
